### PR TITLE
Cedric/implement tf question

### PR DESCRIPTION
**Pull request by cedricongjh**

### CODE REVIEW

File: src/main/java/seedu/smartnus/logic/commands/questions/AddTfCommand.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            + PREFIX_OPTION + "OPTION "
+            + PREFIX_ANSWER + "ANSWER "
+            + PREFIX_IMPORTANCE + "IMPORTANCE "
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_QUESTION + "What is 1+1? "
+            + PREFIX_OPTION + "3 "
+            + PREFIX_OPTION + "4 "
+            + PREFIX_OPTION + "5 "
+            + PREFIX_ANSWER + "2 "
+            + PREFIX_IMPORTANCE + "1 ";
```
Comment (aryagiat) [inline src/main/java/seedu/smartnus/logic/commands/questions/AddTfCommand.java:25]: I think the message usage is not updated to fit T/F questions

**Review by aryagiat — CHANGES_REQUESTED** [review]
Great T/F question implementation. I added a comment [here](https://github.com/AY2122S1-CS2103T-F12-1/tp/pull/63/files#r733320427). And please update the code such that it passes all the checkstyle tests. Other than that, it can be merged 👍

File: src/main/java/seedu/smartnus/logic/commands/questions/AddTfCommand.java (hunk 1 above)
Comment (cedricongjh) [reply src/main/java/seedu/smartnus/logic/commands/questions/AddTfCommand.java]: Updated message usage

File: src/main/java/seedu/smartnus/logic/commands/quiz/AnswerTfCommand.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    public CommandResult execute(Model model) {
+        // TODO: Some additional logic here for the UI
+        requireNonNull(model);
+
+        ArrayList<Choice> choices = currentQuestion.getOrderedChoices();
+        Choice choice = null;
+        switch (input) {
+
+            case "a":
+            case "A":
```
[the remark this answers is not in the document]
Comment (cedricongjh) [reply src/main/java/seedu/smartnus/logic/commands/quiz/AnswerTfCommand.java]: good point, I have made it so that the user answers t/T or f/F instead

**Review by Agentum07 — APPROVED** [review]
LGTM!
I think you have a couple of merge conflicts now that Arya has implemented the theme change feature.
Other than that, it's good to go!

**Review by aryagiat — APPROVED** [review]
LGTM! Thanks for the fix

**Pull request merged**